.github/workflows/snapshot-timestamp.yml

```diff
@@ -43,21 +43,21 @@ jobs:
       - name: setup
         run: |
           echo "REPO=$(pwd)/repository/" >> $GITHUB_ENV
-          echo "SNAPSHOT_KEY=gcpkms://projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/snapshot" >> $GITHUB_ENV
-          echo "TIMESTAMP_KEY=gcpkms://projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/timestamp" >> $GITHUB_ENV
+          echo "SNAPSHOT_KEY=gcpkms://projects/sigstore-root-signing/locations/global/keyRings/root/cryptoKeys/snapshot" >> $GITHUB_ENV
+          echo "TIMESTAMP_KEY=projects/sigstore-root-signing/locations/global/keyRings/root/cryptoKeys/timestamp" >> $GITHUB_ENV
       - uses: actions/setup-go@268d8c0ca0432bb2cf416faae41297df9d262d7f # v2.2.0
         with:
           go-version: 1.18.x
       - uses: google-github-actions/setup-gcloud@877d4953d2c70a0ba7ef3290ae968eb24af233bb # v0.5.1
         with:
-          project_id: project-rekor
+          project_id: sigstore-root-signing
```

Comment:
can we make this change yet? don't we need a new root signing event since we're switching to a new key?

Reply:
the new root signing event is happening today! so i need these triggers